Source/WebKit/WebProcess/Inspector/WebInspectorClient.cpp

```diff
@@ -103,22 +103,22 @@ void WebInspectorClient::frontendCountChanged(unsigned count)
 Inspector::FrontendChannel* WebInspectorClient::openLocalFrontend(InspectorController* controller)
 {
     if (RefPtr page = m_page.get())
-        page->inspector()->openLocalInspectorFrontend();
+        page->protectedInspector()->openLocalInspectorFrontend();
     return nullptr;
 }
 
 void WebInspectorClient::bringFrontendToFront()
 {
     RefPtr page = m_page.get();
-    if (page && page->inspector())
-        page->inspector()->bringToFront();
+    if (page && page->protectedInspector())
+        page->protectedInspector()->bringToFront();
```

Comment:
```suggestion
    if (!page)
        return;

    if (RefPtr inspector = page->inspector();
        inspector->bringToFront();
```
Is what we did elsewhere for cases like this. You should definitely never need a protected member just for a boolean output.